hms-app/src/common/types.ts

```diff
@@ -10,6 +10,13 @@ export enum HackathonDetailsType {
   FullInfo = 'FULLINFO',
 }
 
+export enum IdeaCardTypes {
+  Owner = 'OWNER',
+  Admin = 'ADMIN',
+  Voting = 'VOTING',
+  Other = 'OTHER',
```

Comment:
other seems a bit too... all inclusive.  We could probably be a bit more specific